apps/Webbrowser/qml/Webengine.qml

```diff
@@ -43,7 +42,8 @@ Item {
             text: webengine.url
 
             onAccepted: addressBarTextEntered(text)
-            onFocusChanged: focus ? Qt.inputMethod.show() : Qt.inputMethod.hide()
+            onFocusChanged: focus ? Qt.inputMethod.show(
+                                        ) : Qt.inputMethod.hide()
```

Comment:
that's some auto-formatting? Looks odd.

Reply:
Yes, I tried enabling the qml auto-formatting when saving files in QtCreator. It's not too bad, but it has some glitches...